migrations/20240601122746_keyword_name_kwType_name_and_sound_audio_are_unique__movie_description_expanded/migration.sql

```diff
@@ -0,0 +1,19 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[name]` on the table `Keyword` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[name]` on the table `KeywordType` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[audio]` on the table `Sound` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE `movie` MODIFY `description` TEXT NULL;
```

Comment:
This should be `Movie` with a capital 'M'.

Reply:
Aah yes, I remember that this is an issue when we create migrations right? I can just alter that in the migration file itself right? I don't need to make a new migration, right?

Reply:
I think I can just change the migration file, so I'll just do that!